app/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
   compileSdkVersion 28
   defaultConfig {
     applicationId "com.example.supercookie"
-    minSdkVersion 21
+    minSdkVersion 24
```

Comment:
I didn't dig for _why_ the minimum jumped... if we think it's important to support at 21, I can investigate.

Reply:
I don't see a reason why it went up. We should probably keep it at 21